app/services/sign_in/assertion_validator.rb

```diff
@@ -94,10 +117,26 @@ def user_identifier
       @user_identifier ||= decoded_assertion.sub
     end
 
+    def issuer
+      @issuer ||= decoded_assertion.iss
+    end
+
+    def assertion_audience
+      @assertion_audience ||= decoded_assertion.aud
+    end
+
     def audience
       @audience ||= service_account_config.access_token_audience
     end
 
+    def iat
```

Comment:
Since we're not transmitting these variables, there's no need to be concise, so we should probably name this 'issued_at_time' to make it easier to understand instead of the JWT name for it